ipalib/cli.py

```diff
@@ -1347,9 +1348,7 @@ def run(api):
     try:
         (_options, argv) = api.bootstrap_with_global_options(context='cli')
 
-        if not hasattr(api.env, 'server'):
-            logger.error('IPA is not configured on this system.')
-            return
+        check_client_configuration()
```

Comment:
Hi @flo-renaud,

I've changed the code to use the same check that ipa-client-automount does, but then the ipa help commands does not work anymore.